source/com/vasileff/ceylon/model/ClassAlias.ceylon

```diff
@@ -41,4 +40,9 @@ class ClassAlias(
     shared actual
     Boolean canEqual(Object other)
         =>  other is ClassAlias;
+
+    shared actual Class? refinedDeclaration {
+        assert(is Class? c = Declaration.refinedDeclaration(this));
```

Comment:
Can you make these `super.refinedDeclaration` instead? I'm pretty sure I didn't optimize this on Dart, so this would create a closure.